webroot/results/admin/persons_finish_unfinished_ACTION.php

```diff
@@ -125,6 +141,30 @@ function insertPerson( $newName, $newCountry, $newId ) {
   
   #--- Execute the command.
   dbCommand( $command );
+
+  if ( $oldCountry <> $newCountry || $oldName <> $newName){
+
+    #--- Guess countries iso2
+    $newCountryIso2 = getIso2FromCountryId( $newCountry );
+    if ( $newCountryIso2 ){
```

Comment:
Yes exactly.  The problem is that in this step not only the newcomer's name is editable, but also the country (!?).  The Results Team NEVER edit countries at this point, but just in case.